microsoft/testsuites/dpdk/dpdksuite.py

```diff
@@ -81,28 +68,79 @@ def _execute_expect_zero(self, node: Node, cmd: str) -> str:
         result.assert_exit_code()
         return result.stdout
 
-    def _hugepages_init(self, node: Node) -> None:
-        mount = node.tools[Mount]
-        mount.mount(disk_name="nodev", point="/mnt/huge", type="hugetlbfs")
-        mount.mount(
-            disk_name="nodev",
-            point="/mnt/huge-1G",
-            type="hugetlbfs",
-            options="pagesize=1G",
-        )
+    @TestCaseMetadata(
+        description="""
+           Tests a more realistic sender/forwarder/receiver setup.
+        """,
+        priority=2,
+        requirement=simple_requirement(
+            min_core_count=8,
+            min_nic_count=2,
+            network_interface=Sriov,
+            min_count=3,
+        ),
+    )
+    def check_3node_forwarding(self, environment: Environment, log: Logger) -> None:
+        # TODO: stub test, just submitting refactoring PR
+        pass
```

Comment:
How about move this test case to next PR. So we can merge other parts firstly.

Reply:
sure